packages/client/src/client/muster.client.ts

```diff
@@ -24,4 +28,16 @@ export class MusterClient {
     });
   }
 
+  static getMusterComplianceByDate(orgId: number, unitName: string, query: GetMusterComplianceByDateQuery): Promise<GetMusterComplianceByDateResponse> {
+    return client.get(`${orgId}/${unitName}/complianceByDate`, {
+      params: query,
+    });
+  }
+
+  static getMusterComplianceByDateRange(orgId: number, unitName: string, query: GetMusterComplianceByDateRangeQuery): Promise<GetMusterComplianceByDateRangeResponse> {
```

Comment:
By looking at the **GetMusterComplianceByDateRangeQuery** type I can't tell how to construct the **query** parameter.  Since this is a range, would it be more clear to pass in an object with a start and end date instead of a single string? 
{ isoStartDate: string; isoEndDate: string }

Reply:
I tried to indicate by the name that it is an `isoDateRange` which is a standardized format `YYYY-MM-DDT00:00:00/YYYY-MM-DDT00:00:00`, but perhaps I should use a start and end as part of the query params to be more explicit as opposed to body params? I think this pattern is used for other GET endpoints that want a date range.

Reply:
I personally feel that joining two pieces of data into a single string, even if related, is not the best idea. The names you used are great, no issue there.

Reply:
resolved with query params now